### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by danexpc**

### CODE REVIEW

**Comment by Danil2019IT** [comment]
@nechaido @bugagashenkj

File: lab_work_#7/lab_7.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import random
+
+def array_creation(n, a, b):
+    array = [random.randint(a, b) for i in range(n)]
+    return array
+
+def dictionary_creation(array):
+    dictionary = {}
+    for i in arr:
```
Comment (ivan-tymoshenko) [inline lab_work_#7/lab_7.py]:
```suggestion
    for i in array:
```

File: lab_work_#7/lab_7.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+dictionary = dictionary_creation(arr)
+del arr
+
+popular_number = appear_the_most(dictionary, b)
+print(F'Наиболее встречаемое число -- {popular_number}')
+
+
+
+
+
```
Comment (ivan-tymoshenko) [inline lab_work_#7/lab_7.py]: Unnecessary lines

File: lab_work_#7/lab_7.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+n = int(input("Введите размер массива: "))
+
+print("Задайте диапазон целых чисел в массиве: ")
+a = int(input("От: "))
+b = int(input("До: "))
+arr = array_creation(n, a, b)
+print(arr)
+
+dictionary = dictionary_creation(arr)
+del arr
```
Comment (ivan-tymoshenko) [inline lab_work_#7/lab_7.py]:
```suggestion
```

File: lab_work_#8/lab_8.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+print(F"Y = {value_Y(B, T)}")
+
+
+
+
+
+
+
+
```
Comment (ivan-tymoshenko) [inline lab_work_#8/lab_8.py]: Unnecessary lines

**Comment by danexpc** [comment]
@bugagashenkj

**Comment by danexpc** [comment]
@bugagashenkj

**Review by ivan-tymoshenko — APPROVED** [review]

**Pull request merged**